src/DocusignDriverServiceProvider.php

```diff
@@ -16,6 +17,10 @@ public function boot(): void
         $this->publishes([
             __DIR__.'/../config/docusigndriver.php' => config_path('docusigndriver.php'),
         ]);
+
```

Comment:
![CleanShot 2024-02-14 at 07 13 24@2x](https://github.com/alnutile/docusign_driver/assets/365385/e6d36b34-3ded-4bc0-a0d7-310a11eec527)

Reply:
If there is a problem doing it the way they suggest let me know

Reply:
done